webapp/components/navbar.jsx

```diff
@@ -212,16 +219,32 @@ export default class Navbar extends React.Component {
         this.setState({showMembersModal: false});
     }
 
-    showChannelSwitchModal(e) {
-        if (Utils.cmdOrCtrlPressed(e) && e.keyCode === Constants.KeyCodes.K) {
+    handleQuickSwitchKeyPress(e) {
+        if (Utils.cmdOrCtrlPressed(e, true) && e.keyCode === Constants.KeyCodes.K) {
             e.preventDefault();
-            this.setState({showChannelSwitchModal: !this.state.showChannelSwitchModal});
+            if (e.altKey) {
+                if (getMyTeams(store.getState()).length <= 1) {
+                    return;
+                }
+                this.showQuickSwitchModal('team');
+            } else {
+                this.showQuickSwitchModal('channel');
+            }
+        }
+    }
+
+    showQuickSwitchModal(mode = 'channel') {
```

Comment:
Can you rename this to `toggleQuickSwitchModal` to better reflect what it actually does?